tools/taskcluster-proxy/README.md

```diff
@@ -0,0 +1,247 @@
+# taskcluster-proxy
+
+taskcluster-proxy is the proxy server which is used by Taskcluster workers to
+enable individual tasks to talk to various Taskcluster services (auth, queue,
+scheduler, ...) without hardcoding credentials into the containers themselves.
+
+* When used by docker-worker the taskcluster-proxy runs in a separate docker
+  container linked to the task docker container.
+* When used by generic-worker, the taskcluster-proxy runs as a separate
+  native executable on the host.
+
+## Download binary release
+
+You do __not__ need to install go (golang) to run the proxy. It is shipped as a
+native executable.
+
+See [releases page](https://github.com/taskcluster/taskcluster-proxy/releases)
+and choose a download that matches your platform.
+
+## Download source and install via `go get`
+
+Alternatively you can build and install from source. For this it is recommended
+you install the latest version of go (golang) first.
+
+```sh
+go get github.com/taskcluster/taskcluster-proxy
+```
```

Comment:
I think this should be `github.com/taskcluster/taskcluster/v<latest>/tools/taskcluster-proxy` where `<latest>` is the latest release. We should check this `README.md` will get updated with a find/replace regexp like other files in the repo when a new release is made, so this reference gets updated automatically.

Reply:
> We should check this README.md will get updated with a find/replace regexp like other files in the repo when a new release is made, so this reference gets updated automatically.

Where do I check that?

Reply:
The regexp is run over all of `tools/`, so it should be fine.